src/frontend/Komponenter/Behandling/Vurdering/VisVurdering.tsx

```diff
@@ -60,7 +59,7 @@ interface Props {
     vurdering: IVurdering;
     resetVurdering: () => void;
     feilmelding: string | undefined;
-    settRedigeringsmodus: (redigeringsmodus: Redigeringsmodus) => void;
+    settRedigeringsmodus: () => void;
```

Comment:
Burde denne endre navn ettersom den ikke lenger tar inn en funksjon for å sette en state? Syns egentlig det, men kom ikke på noe navn. Forslag tas i mot med stor takk🤗

Reply:
`startRedigering` kanskje?